README.md

```diff
@@ -25,9 +25,24 @@ These six steps are as follows:
 
 Current p-Creode methodology entails running steps one through five 100 times and then running step 6 one time. Each of the six steps can be found within the [_functions.py_](https://github.com/KenLauLab/pCreode/blob/master/pcreode/functions.py) file.
 
-### Installation for Mac or Linux
+### Installation for Linux
 
-There are three ways to install p-Creode with Mac/Linux operating systems.
+1. Install [Anaconda](https://www.anaconda.com/products/individual)
+
+2. Run the following in `Terminal`:
+
+```cmd
+git clone git://github.com/KenLauLab/pCreode
+cd pCreode
+conda create -n pcreode python=3.7 numpy pandas matplotlib python-igraph jupyter cython -c conda-forge
```

Comment:
If this errors out with python-igraph not found, should add another line.

` conda install -c conda-forge python-igraph `

Reply:
is it erroring out for you? the `-c conda-forge` flag at the end of the command should include conda-forge if it can't find any libs in the base location

Reply:
Just tried, it works well. No need for extra line.